labelbox/data/serialization/coco/instance_dataset.py

```diff
@@ -221,8 +236,13 @@ def to_common(self, image_root):
                             class_annotations, category_lookup[
                                 class_annotations.category_id].name))
                 elif isinstance(class_annotations.segmentation, list):
-                    annotations.extend(
-                        segmentations_to_common(
+                    if len(class_annotations.segmentation) == 0 and len(class_annotations.bbox) == 4:
```

Comment:
Is `class_annotations.bbox` ever anything other than a list with 4 items? Is this check required?

Reply:
Actually, I'm wrong; It doesn't clearly convey that at all but I still think it's better than testing the length.

Reply:
Should I include both in case there is an iterator but has no elements?

Reply:
Actually I'd leave it as it is. It's such a minor thing it's not worth changing.